autogpt/commands/file_operations.py

```diff
@@ -179,6 +194,59 @@ def append_to_file(filename: str, text: str, should_log: bool = True) -> str:
         return f"Error: {str(e)}"
 
 
+@command(
+    "update_file",
+    "Update file",
+    '"filename": "<filename>", '
+    '"old_text": "<old_text>", "new_text": "<new_text>", '
+    '"match_index": "<match_index>"',
```

Comment:
This has to be renamed to a more descriptive name like `replace_occurrence_index`, and indicated as an optional int parameter

Reply:
On it